third_party/akri/update-akri.sh

```diff
@@ -3,48 +3,9 @@
 # fetch the latest version
 helm repo add akri-helm-charts https://project-akri.github.io/akri/ 
 helm repo update akri-helm-charts
-helm pull akri-helm-charts/akri --untar
+helm pull akri-helm-charts/akri
```

Comment:
would it make sense to pin a version here?

Reply:
Good idea. Done.